packages/store/src/usersSlice/thunks.test.ts

```diff
@@ -344,15 +342,17 @@ describe('user actions', () => {
       { type: editUser.fulfilled.type }
     ];
     const store = mockStore({ ...defaultState });
-    await store.dispatch(editUser({ id: 'a1', email: defaultState.users.byId.a1.email, password: defaultPassword }));
+    await store.dispatch(editUser({ id: 'a1', email: defaultState.users.byId.a1.email, password: defaultPassword, current_password: 'current_password' }));
```

Comment:
I guess we could extend the corresponding msw handler with a check for the `current_password` to ensure the rejection

Reply:
please, create a ticket

Reply:
https://northerntech.atlassian.net/browse/QA-984